contracts/ICO/EasyMineIco.sol

```diff
@@ -159,35 +150,32 @@ contract EasyMineIco {
     atStage(Stages.AuctionSetUp)
   {
     // Start allowed minimum 3000 blocks from now
-    if (block.number + 3000 >= _startBlock) {
-      revert();
-    }
+    require(_startBlock > block.number + 3000);
+
     startBlock = _startBlock;
     endBlock = startBlock + MAX_DURATION_BLOCKS;
     stage = Stages.AuctionStartScheduled;
   }
 
   /* Submits a pre ICO bid */
   function preIcoBid(address _receiver)
-    public
+    external
     payable
     isPreIco
     timedTransitions
     atStage(Stages.AuctionStartScheduled)
   {
     uint256 amount = msg.value;
-    if (_receiver == 0 || amount == 0) {
-      revert();
-    }
+    require(_receiver != 0x0);
+    require(amount != 0x0);
+
     if (!wallet.send(amount)) {
```

Comment:
Change to 

assert(wallet.send(amount)) for more consistence require/assert over revert/throw semantics usage